.github/workflows/deploy-gh-pages.yml

```diff
@@ -41,12 +41,12 @@ jobs:
         run: npm ci --legacy-peer-deps
 
       - name: Build Next.js Project
-        run: npm run build
+        run: npm run export
```

Comment:
Is this correct? Looking at 
https://github.com/EnAccess/OpenPAYGO-js/blob/main/.github/workflows/deploy-gh-pages.yaml#L46-L47
we are using `build` here